boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -218,10 +218,98 @@ However, we need to short 2 pins on Host Connector J3 via a 3.3k resistor
 (simple shorting without the resistor will also do) for debugging to work.
 Those 2 pins are Pin5 HOST_RST_N_LT_R) and Pin21 (+V_HOST_3P3_1P8).
 
+MCUbootloader Support
+=====================
+
+Intel S1000 on Zephyr supports the open source, cross-RTOS MCUbootloader.
+MCUbootloader in conjuntion with USB DFU (Device Firmware Upgrade)
+over the air can be used for Field upgradation of Intel S1000 Firmware.
+The MCUbootloader source code itself is hosted in the
+`MCUboot GitHub repo`_ page. Check out `MCUboot with Zephyr`_ documentation
+page on the MCUboot website for more on MCUboot.
+
+First, you need to generate the MCUbootloader image for Intel S1000.
+Go to the mcubootloader repository on your local machine and execute the
+below commands.
+
+.. code-block:: console
+
+   cd boot/zephyr
+   mkdir build
+   cd build
+   cmake -DBOARD=intel_s1000_crb ..
+   make
+
+zephyr/zephyr.bin is the output of the above steps and this needs to be passed
+through a utility 'create_spi_flash' which will pad this binary suitably so
+that this gets recognized by the S1000 BootROM. Below is the command.
+
+.. code-block:: console
+
+   ./create_spi_flash zephyr/zephyr.bin mcuboot.bin -sk
```

Comment:
create_spi_flash utility is not available publicly. need to make it or a variant of it as part of the build system before updating the document.

Reply:
ok. noted. I will keep this for the time being while i work on the utility

Reply:
Removed this now. https://github.com/zephyrproject-rtos/zephyr/pull/12662 addresses this.